.gitignore

```diff
@@ -23,3 +23,4 @@ endpoints.json
 generate-doc
 # Ignore vi backup files
 *~
+.idea
```

Comment:
Is this required in this PR?

Reply:
No, .idea file created by IDE(IntelliJ), So just to ignore those file, I have added it in .gitignore

Reply:
this is specific to IDE, people use different development environment. I  think we don't need to add here

Reply:
This is okay. Lot of projects do this. `*~` and `.swp` are vi/vim specific files. I've seen projects with Mac [specific](https://en.wikipedia.org/wiki/.DS_Store) file references in `.gitreview.`

Reply:
See https://github.com/heketi/heketi/blob/master/.gitignore too :)